tests/functional-tests/README.md

```diff
@@ -0,0 +1,77 @@
+# IBM Spectrum Scale CSI - Automation
+
+The IBM Spectrum Scale Container Storage Interface (CSI) project enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage.
+
+This project contains an automation framework to exercise the IBM Spectrum Scale CSI Driver and Operator functionality .
+
+
+### Testbed environment 
+
+- IBM Spectrum Scale Cluster - 5.0.4.1+ Version  (**IBM Spectrum Scale supported kernel version**)
+- Kubernetes Cluster Version 1.14 - 1.18
+- Openshift Version 4.2.x,4.3.x
+- IBM Spectrum Scale Cluster CSI Version - 1.1.0+
```

Comment:
Is this the correct place to put these versions? 

If it's itemizing the environments that we have verified on, shouldn't it be higher up on the repo level? 

Is it possible to create some of this info without the need to continually update this on every release... maybe it's ok.. just thinking out loud.